### DISCUSSION TITLE
Stuck at Installing various dependencies

### DISCUSSION THREAD

**Discussion by myxr6turbo** [Security-Onion-Solutions/securityonion#1744] [Unsupported Versions]
Hi guys,

I have created a ESXi VM with 250gb Thick VM and after is asks all the questions it starts to install and stops at 6% "Installing various dependencies". I can SSH to it so it's getting an IP. I have checked the hash of the ISO file and it's correct. I even downloaded it again just to make sure. I have tried both Production and Eval. same problem. Any help would be much appreciated.

**Comment by abs0248** [comment]
> Hi guys,
> 
> I have created a ESXi VM with 250gb Thick VM and after is asks all the questions it starts to install and stops at 6% "Installing various dependencies". I can SSH to it so it's getting an IP. I have checked the hash of the ISO file and it's correct. I even downloaded it again just to make sure. I have tried both Production and Eval. same problem. Any help would be much appreciated.

I had same problem and it look like the installation need the machine have internet access. Without internet access it take a very long time to finish.
